src/test/java/bio/terra/grammar/GrammarTest.java

```diff
@@ -0,0 +1,57 @@
+package bio.terra.grammar;
+
+
+import bio.terra.common.category.Unit;
+import bio.terra.grammar.exception.InvalidQueryException;
+import bio.terra.grammar.google.BigQueryVisitor;
+import bio.terra.model.DatasetModel;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+@Category(Unit.class)
+public class GrammarTest {
+
+    private Map<String, DatasetModel> datasetMap;
+
+    @Before
+    public void setup() {
+        DatasetModel datasetModel = new DatasetModel().dataProject("a-data-project");
+        datasetMap = Collections.singletonMap("dataset", datasetModel);
+    }
+
+    @Test
```

Comment:
I'd also be interested to see a test that triggers failures (invalid query, invalid results, etc)

Reply:
Thanks for adding those tests!